dashboard/legacy/middleware/helpers/projects.rb

```diff
@@ -362,7 +362,7 @@ def most_recent_project_type(type)
   # 'createdAt', 'updatedAt', 'publishedAt' and 'projectType' merged from the
   # corresponding database row values.
   def self.merged_row_value(row, channel_id:, is_owner:)
-    JSON.parse(row[:value]).merge(
+    JSON.parse(row[:value], symbolize_names: true).merge(
```

Comment:
There are 2 unit tests that are failing in `test_projects.rb`. I can fix them by:
- accessing thumbnailUrl with symbol in `test_project_create_succeeds_with_valid_thumbnail_url`
- for `test_derive_project_type` I need to change the order of the two hashes that are being merged in merged_row_value in projects.rb. https://github.com/code-dot-org/code-dot-org/blob/ec6d13f6bd3c5e63569074303dc9d04260fad2a2/dashboard/legacy/middleware/helpers/projects.rb#L364

Making further change in a legacy file is making me nervous so I'm going to revert PR to draft and discuss a bit further with folks..

Reply:
Discussed with @cnbrenci and we agreed to keep changes as minimal as possible.